brave/brave-core
# Adds functionality to monthly contribution dropdown (Panel)

Fixes: https://github.com/brave/brave-browser/issues/2245

The brunt of the work for this was adding extension functions to support:

Adding a recurring contribution
Removing a recurring contribution
Retrieving recurring contributions

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
- [] Ran `git rebase master` (if needed).

## Test Plan:
   1. Enable Brave Rewards
   2. Navigate to a site such as `brave.com`
   3. Bring up the wallet panel.
   4. Change monthly contribution amount to '5.0'
   5. Close and re-open panel
   6. Confirm that the value '5.0' persists
   7. Confirm that the change is reflected in the donation table
 
   The above six steps should be tried in different ways with different values. Ex: Ensuring that the data persists across tabs with the same site open

## Reviewer Checklist:
